opensearchpy/connection/http_async.pyi

```diff
@@ -6,6 +6,23 @@
 #
 # Modifications Copyright OpenSearch Contributors. See
 # GitHub history for details.
+#
```

Comment:
These licenses headers are not required because this file was created post fork.

Reply:
Made the change suggested.